Make exercise guides available in English (#489)
dodona-edu/dodona-edu.github.io: en/guides/exercises/creating-exercises/management/index.md

---
title: "Creating exercises: next steps"
sidebarTitle: Next steps
order: 4
prev: true
next: false
---

# Creating Exercises: Next Steps

Now that you have an exercise that can be used on Dodona, there are many additional things you can do.
Here are some suggestions for your next steps:

- [Create a course](/en/guides/teachers/creating-a-course/).
- [Use your exercise in an exercise series within a course](/en/guides/teachers/exercise-series-management/).
- Create [reading activities](/en/guides/exercises/examples/content/) with extra information or small pieces of theory.
- Develop other types of exercises.

> Enjoy using Dodona!

Reviewer: ```suggestion
Have fun using Dodona!
```

## More About Test Suites

There are various resources available for working with test suites:

> - Check out the [guide with common scenarios](/en/guides/exercises/testsuites). Often, what you need is already covered there.

Reviewer: ```suggestion
- Check out the [guide with common scenarios](/en/guides/exercises/testsuites); it probably has what you're looking for
```
